pkg/config/autodiscover/autodiscover_targets.go

```diff
@@ -35,6 +35,8 @@ const (
 	skipConnectivityTestsLabel  = "skip_connectivity_tests"
 	ocGetClusterCrdNamesCommand = "kubectl get crd -o json | jq '[.items[].metadata.name]'"
 	DefaultTimeout              = 10 * time.Second
+	Statefullset                = "statefullset"
```

Comment:
This is a bit of a nitpick, but the right spelling is "Statefulset". I see it is in multiple places in the PR.